packages/boot/test/unit/bootstrapper.unit.ts

```diff
@@ -13,49 +13,50 @@ describe('boot-strapper unit tests', () => {
   let app: BootApp;
   let bootstrapper: Bootstrapper;
   const booterKey = `${BootBindings.BOOTER_PREFIX}.TestBooter`;
-  const booterKey2 = `${BootBindings.BOOTER_PREFIX}.TestBooter2`;
+  const anotherBooterKey = `${BootBindings.BOOTER_PREFIX}.AnotherBooter`;
 
   beforeEach(getApplication);
   beforeEach(getBootStrapper);
 
   it('finds and runs registered booters', async () => {
     const ctx = await bootstrapper.boot();
-    const booterInst = await ctx.get(booterKey);
-    expect(booterInst.configureCalled).to.be.True();
-    expect(booterInst.loadCalled).to.be.True();
+    const booterInst = await ctx.get<TestBooter>(booterKey);
+    expect(booterInst.phasesCalled).to.eql([
+      'TestBooter:configure',
+      'TestBooter:load',
+    ]);
   });
 
   it('binds booters passed in BootExecutionOptions', async () => {
-    const ctx = await bootstrapper.boot({booters: [TestBooter2]});
-    const booterInst2 = await ctx.get(booterKey2);
-    expect(booterInst2).to.be.instanceof(TestBooter2);
-    expect(booterInst2.configureCalled).to.be.True();
+    const ctx = await bootstrapper.boot({booters: [AnotherBooter]});
+    const anotherBooterInst = await ctx.get<AnotherBooter>(anotherBooterKey);
+    expect(anotherBooterInst).to.be.instanceof(AnotherBooter);
+    expect(anotherBooterInst.phasesCalled).to.eql(['AnotherBooter:configure']);
   });
 
   it('no booters run when BootOptions.filter.booters is []', async () => {
     const ctx = await bootstrapper.boot({filter: {booters: []}});
-    const booterInst = await ctx.get(booterKey);
-    expect(booterInst.configureCalled).to.be.False();
-    expect(booterInst.loadCalled).to.be.False();
+    const booterInst = await ctx.get<TestBooter>(booterKey);
+    expect(booterInst.phasesCalled).to.eql([]);
   });
 
   it('only runs booters passed in via BootOptions.filter.booters', async () => {
     const ctx = await bootstrapper.boot({
-      booters: [TestBooter2],
-      filter: {booters: ['TestBooter2']},
+      booters: [AnotherBooter],
+      filter: {booters: ['AnotherBooter']},
     });
-    const booterInst = await ctx.get(booterKey);
-    const booterInst2 = await ctx.get(booterKey2);
-    expect(booterInst.configureCalled).to.be.False();
-    expect(booterInst.loadCalled).to.be.False();
-    expect(booterInst2.configureCalled).to.be.True();
```

Comment:
These assertions are rather unhelpful when they fail:

```
  1) boot-strapper unit tests
       only runs booters passed in via BootOptions.filter.booters:
     AssertionError: expected false to be true
      at Assertion.fail (packages/testlab/node_modules/should/as-function.js:275:17)
      at Assertion.value (packages/testlab/node_modules/should/as-function.js:356:19)
      at Context.it (packages/boot/test/unit/bootstrapper.unit.ts:4:65)
      at <anonymous>
```

While I was fixing the cause of the test failure, I rewrote all tests to be more helpful when failing:

```
     AssertionError: expected Array [ ] to equal Array [ 'TestBooter:configure' ]
      + expected - actual

       [
      +  "TestBooter:configure"
       ]
```